sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/Analytics.java

```diff
@@ -243,14 +257,60 @@ public static void trackEvent(String name) {
      */
     @SuppressWarnings("WeakerAccess")
     public static void trackEvent(String name, Map<String, String> properties) {
+        trackEvent(name, properties, null);
+    }
+
+    /**
+     * Track a custom event with name and transmissionTarget.
+     *
+     * @param name A page name.
+     * @param transmissionTarget The transmissionTarget for this event.
+     */
+    @SuppressWarnings({"WeakerAccess", "SameParameterValue"})
+    public static void trackEvent(String name, TransmissionTarget transmissionTarget) {
+        trackEvent(name, null, transmissionTarget);
+    }
+
+    /**
+     * Track a custom event with name and optional properties and optional transmissionTarget.
+     * The name parameter can not be null or empty. Maximum allowed length = 256.
+     * The properties parameter maximum item count = 5.
+     * The properties keys can not be null or empty, maximum allowed key length = 64.
+     * The properties values can not be null, maximum allowed value length = 64.
+     * Any length of name/keys/values that are longer than each limit will be truncated.
+     *
+     * @param name       An event name.
+     * @param properties Optional properties.
+     * @param transmissionTarget Optional transmissionTarget.
+     */
+    @SuppressWarnings("WeakerAccess")
+    public static void trackEvent(String name, Map<String, String> properties, TransmissionTarget transmissionTarget) {
         final String logType = "Event";
         name = validateName(name, logType);
         if (name != null) {
             Map<String, String> validatedProperties = validateProperties(properties, name, logType);
-            getInstance().trackEventAsync(name, validatedProperties);
+            getInstance().trackEventAsync(name, validatedProperties, transmissionTarget);
         }
     }
 
+    public static TransmissionTarget getTransmissionTarget(String transmissionTargetToken) {
```

Comment:
move to getInstance().getSomething and use synchronized at instance level like everywhere else